README.md

```diff
@@ -30,8 +31,8 @@ with the JSON Schema model.
 ```javascript
 var compile = require('protobuf-jsonschema');
 
-var all = compile('models.proto');
-var single = compile('models.proto', 'MyModel');
+var all = compile('models.proto', true);
+var single = compile('models.proto', 'MyModel', true);
```

Comment:
Can we make the API `compile(filename, model, opts)` where `opts` is an object `{additionalProperties: true}`.